files/wishlist.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wishlist</title>
+    <link href="https://anti-strap-ui.netlify.app/app.css" rel="stylesheet" />
+    <link href="../app.css" rel="stylesheet" />
+  </head>
+  <body>
+    <nav class="navbar nav">
+        <div>
+            <a href="/index.html" class="nav-head-link"><h2>The Chai Store</h2></a>
+        </div>
+        <div class="nav-search-bar">
+            <input class="input-text" style="font-family: FontAwesome;" placeholder='&#xf002 Search...' />
+        </div>
+        <div class="nav-link-container">
+            <a href="index.html" class="btn btn-primary-outline navigation-link">Login</a>
+            <a href="#" class="navigation-link"><div class="badges">
+                <i class="fas fa-heart nav-cart"></i>
+                <span class="badge">2</span>
+            </div></a>
+            <a href="#" class="navigation-link"><div class="badges">
+                <i class="fas fa-shopping-cart nav-cart"></i>
+                <span class="badge">2</span>
+            </div></a>
+        </div>
+        <div class="nav-hamburger"> <i class="fas fa-bars "></i></i></div>
+        </div>
+    </nav>
+    <main>
+      <h1 class="wishlist-display-title">My Wishlist</h1>
+      <section class="wishlist-card-container">
```

Comment:
card should have a dismiss icon/badge so the user can remove it from wishlist.

Reply:
It has a heart icon so when it click it removes